src/map/handler/Map.Keyboard.js

```diff
@@ -162,7 +163,7 @@ export var Keyboard = Handler.extend({
 		} else if (key in this._zoomKeys) {
 			map.setZoom(map.getZoom() + (e.shiftKey ? 3 : 1) * this._zoomKeys[key]);
 
-		} else if (key === 27 && map._popup) {
+		} else if (key === 27 && mapPopup && mapPopup.options.closeOnEscapeKey) {
```

Comment:
Not sure we need an intermediate variable to save out 2 chars on two occurrences ;)

Reply:
Change made 🙂